src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js

```diff
@@ -110,20 +106,13 @@ const GlobalHeader = ({ className, activeSite }) => {
         css={[
           css`
             position: relative;
-            z-index: 1;
+            top: 0;
+            z-index: 80;
+            border: 1px solid #83878b;
```

Comment:
The header in the designs only has a border on the bottom. Updating this to use `border-bottom` should solve that.
```suggestion
            border-bottom: 1px solid #83878b;
```

Reply:
@AlecIsaacson Actually I also thought the same but in the figma design they had give the entire border thats why i keep it as a border. I changed the border to border-bottom.